src/mocks/handlers/company.ts

```
@@ -23,6 +23,7 @@ type CompanyInfo = {
 /* =========================
    In-Memory DB
 ========================= */
+let nextNewsId = 1;
```

ESLint

[error] 'nextNewsId' is never reassigned. Use 'const' instead.

(prefer-const)